src/app/lib/utilities/passport/index.js

```diff
@@ -0,0 +1,38 @@
+import pathOr from 'ramda/src/pathOr';
+
+export const getPassportHome = pageData => {
+  const homeServiceUrl = pathOr(
+    null,
+    ['metadata', 'passport', 'home'],
+    pageData,
+  );
+  const passportHomeOverride = pathOr(
+    null,
+    ['metadata', 'passportHome'],
+    pageData,
+  );
+  return (
+    passportHomeOverride ||
+    (pageData && homeServiceUrl
+      ? homeServiceUrl
+          .split('/')
+          .slice(-1)
+          .pop()
+          .toLowerCase()
```

Comment:
What happens if we get back a `home` that doesn't match this format?

Reply:
Is this possible? @dr3 @hindsc52

Reply:
![image](https://user-images.githubusercontent.com/11341355/65047318-31f5ff00-d95a-11e9-80c2-8850cbfebef5.png)
As long as its a string, its fine :)